filter-effects-2/Overview.bs

```diff
@@ -50,28 +53,283 @@ If the value of the 'backdrop-filter' property is ''backdrop-filter/none'' then
 filter effect applied. Otherwise, the list of functions are applied in the
 order provided.
 
-The first filter function or <a element>filter</a> reference in the list takes
-the element's BackgroundImage as the input image. Subsequent
+## Filtering and Clipping ## {#backdrop-filter-operation}
+
+An element (call it B) with a backdrop-filter property other than ''backdrop-filter/none'' is conceptually rendered as if
+the following steps are performed:
+1. Copy the <a>Backdrop Root Image</a> into a temporary buffer, such as a raster image. Call this buffer T’.
+2. Apply the backdrop-filter’s filter operations to the entire contents of T'.
+3. If element B has any transforms, apply the <b>inverse</b> of those transforms to the contents of T’.
+4. Apply a clip to the contents of T’, using the border box of element B, included <a href="https://drafts.csswg.org/css-backgrounds-3/#border-radius">border-radius</a> if specified. Note that the children of B are not considered for the sizing or location of this clip.
+5. Draw all of element B, including its background, border, and any children elements, into T’.
+6. If element B has any transforms, effects, or clips, apply those to T’.
+7. Composite the contents of T’ into element B's parent, using source-over compositing.
+
+<pre class=link-defaults>
+  spec:filter-effects-1; type:element; text:filter
+</pre>
+
+The first <a element>filter</a> function or filter reference in the <<filter-value-list>>
+takes the element's <a>Backdrop Root Image</a> as the input image. Subsequent
 operations take the output from the previous filter function or <a
-element>filter</a> reference as the input image. <a element>filter</a> element
+element>filter</a> reference as the input image. The <a element>filter</a> element
 reference functions can specify an alternate input, but still uses the previous
 output as its SourceGraphic.
 
 Filter functions must operate in the sRGB color space.
 
-A computed value of other than ''backdrop-filter/none'' results in the creation of a <a
-href="https://www.w3.org/TR/CSS21/zindex.html">stacking context</a> [[!CSS21]]
-the same way that CSS 'opacity' does. All the elements descendants are rendered
-together as a group with the filter effect applied to the group as a whole.
+Note: that the effect of the backdrop-filter will not be visible unless some
+portion of element B is semi-transparent. Also note that any opacity applied to
+element B will be applied to the filtered backdrop image as well. Therefore, to
+create a "transparent" element that allows the full filtered backdrop image to
+be seen, you can use "background-color: transparent;".
+
+Note: A computed value of other than ''backdrop-filter/none'' results in the
```

Comment:
This shouldn't be in a note, since it's normative text.

Reply:
Done. I put the other note below this part, also.